### PR TITLE
Feature/directories

### PR DESCRIPTION
**Pull request by cjstevens78**
- For ‘contact’ details that contain a ‘contact name’ and ‘job title’ - styling has been added to format the titles as bold and the content as a para
- For further information links - Add a fallback pseudo icon ‘link chain’ for items without a specific icon class.
- Change made to directories search button so it renders as a button rather than input - this allows the display of the magnifying glass icon

### CODE REVIEW

File: templates/form/input--submit.html.twig [hunk 1]
```diff
@@ -11,12 +11,18 @@
...
 
-{# Header search sumbit #}
+{# Header or Directories search submit #}
 {% set form_input_submit = 'header-search-block-form-submit' %}
+{% set is_dir_channel = attributes.hasAttribute('data-drupal-selector') and attributes['data-drupal-selector']|render starts with 'edit-submit-localgov-directory-channel' %}
 
-{# Changing the header's search submit input element to a button element #}
-{% if form_input_submit in attributes.class %}
+{# Changing the header or directory search submit input element to a button element #}
+
```
Comment (Adnan-cds) [inline templates/form/input--submit.html.twig]: This empty line is not needed really.

File: templates/form/input--submit.html.twig [hunk 2]
```diff
@@ -11,12 +11,18 @@
...
+{% if (form_input_submit in attributes.class) %}
     <button class="btn icon-search" type="submit" value="Search" aria-label="Search this website" title="Search this website"><span class="fa fa-search" aria-hidden="true"></span></button>
+{% elseif (is_dir_channel)%}
```
Comment (Adnan-cds) [inline templates/form/input--submit.html.twig]: [if statements in Twig](https://twig.symfony.com/doc/2.x/tags/if.html) can do without the brackets around the conditions.  This means `{% if (form_input_submit in attributes.class) %}` is the equivalent of the simpler `{% if form_input_submit in attributes.class %}`

File: templates/form/input--submit.html.twig [hunk 3]
```diff
@@ -11,12 +11,18 @@
...
+    <button data-drupal-selector="edit-submit-localgov-directory-channel" type="submit" id="edit-submit-localgov-directory-channel" value="Search" class="button js-form-submit form-submit btn btn-primary btn-sm mt-2" aria-label="Search directories" title="Search directories">
+  <span>Search</span>
+</button>
```
Comment (Adnan-cds) [inline templates/form/input--submit.html.twig]: The indentation isn't looking right here :(

**Review by Adnan-cds — COMMENTED** [review]
The changes work for me as expected.  If you can please review the comments I have added for the Twig changes.

File: templates/form/input--submit.html.twig (hunk 1 above)
Comment (cjstevens78) [reply templates/form/input--submit.html.twig]: fixed

File: templates/form/input--submit.html.twig (hunk 2 above)
Comment (cjstevens78) [reply templates/form/input--submit.html.twig]: fixed

File: templates/form/input--submit.html.twig (hunk 3 above)
Comment (cjstevens78) [reply templates/form/input--submit.html.twig]: fixed

**Review by Adnan-cds — APPROVED** [review]
Looks good.  Thanks a lot :)

**Pull request merged**